crates/astria-proto/proto/astria/sequencer/v1alpha1/account.proto

```diff
@@ -0,0 +1,21 @@
+// Requests for account information on and responses from the astria sequencer.
+
+syntax = "proto3";
+
+package astria.sequencer.v1alpha1;
+
+import "astria/primitive/v1/types.proto";
+
+// A response containing the balance of an account.
+message BalanceResponse {
+  bytes account = 1;
```

Comment:
do we need to return account here since account is already in the request? same for `NonceResponse`